src/services/edusharing/logic/connector.js

```diff
@@ -211,25 +195,32 @@ class EduSharingConnector {
 		};
 
 		const parsed = await this.requestRepeater(options);
+		return this.EduSearchResponse(parsed);
+	}
 
```

Comment:
We don't need to use the accessToken to the images since it doesn't work and doesn't have any impact